src/utilities/password.util.ts

```diff
@@ -0,0 +1,12 @@
+import crypto = require('crypto');
```

Comment:
`import cube from './my-module.js';`

use either CJS/ESM

Reply:
When I use this syntax, I'm getting an error:
`import crypto from 'crypto';
`

The error says "Module '"crypto"' has no default export."